lib/rest/preview/sync/service/document.d.ts

```diff
@@ -25,13 +25,33 @@ import { SerializableClass } from '../../../../interfaces';
  */
 declare function DocumentList(version: Sync, serviceSid: string): DocumentListInstance;
 
+/**
+ * Options to pass to remove
+ *
+ * @property ifMatch - The if_match
+ */
+interface DocumentInstanceDeleteOptions {
+  ifMatch?: string;
+}
+
+/**
+ * Options to pass to remove
+ *
+ * @property ifMatch - The if_match
```

Comment:
Maybe `The if_match header`?

Reply:
So I think the main issue here is that the if_match param in api-definitions doesn't have a matching doc entry. Not sure if the best fix here is to request that header params get descriptions in the docs or if we should just autogenerate `The {param_name} header` descriptions for each of these.

Reply:
We're autogenerating `The {param_name}` right now so thinking changing the autogeneration for headers to be `The {params_name} header` and also make `param_name` be the kebab case header name.

Reply:
fixed!